code/modules/events/disease_outbreak.dm

```diff
@@ -1,3 +1,14 @@
+#define ADV_MIN_SYMPTOMS 3 //Advanced min symptoms
+#define ADV_MAX_SYMPTOMS 4 //Advanced max symptoms
+#define ADV_ANNOUNCE_DELAY 75 //How long the virus stays hidden
+#define ADV_DISEASE_MEDIUM 4
+#define ADV_DISEASE_HARMFUL 5
+#define ADV_DISEASE_DANGEROUS 7
+#define ADV_RNG_LOW 30 //Percentile for low severity
+#define ADV_RNG_MID 85 //Percentile for mid severity
+#define ADV_SPREAD_LOW 30 //Percentile for low transmissibility
+#define ADV_SPREAD_MID 90 //Percentile for mid transmissibility
```

Comment:
What does ADV even stand for? Admin Virus? Just do `ADMIN_VIRUS_***`, won't take up that much space.

Also, DMDoc instead of having the comments be off to the side.

Reply:
advanced disease.

Reply:
I suppose that's fine. Do DMDoc this though.